state/workingset.go

```diff
@@ -165,8 +165,10 @@ func (ws *workingSet) Height() uint64 {
 
 // RunActions runs actions in the block and track pending changes in working set
 func (ws *workingSet) RunActions(
```

Comment:
Can we pass this information as a Context. I could image that we will add more such kind of attributes. For example, https://solidity.readthedocs.io/en/v0.4.24/units-and-global-variables.html#block-and-transaction-properties